### DISCUSSION TITLE
Should reset_password_token_secret/verification_token_secret be rotated ?

### DISCUSSION THREAD

**Discussion by kplatis** [fastapi-users/fastapi-users#1420] [Q&A]
Hello there. I have a question about the secret used in `reset_password_token_secret` and `verification_token_secret`. The way I use it is to generate a strong passphrase and pass it in the application as an environment variable.

However, should the secret be rotated/changed regularly ?

**Comment by frankie567** [comment]
I would say yes, changing it regularly is a good practice. That said, bear in mind that existing tokens will stop working once your secret is changed.
